src/Context/Context.php

```diff
@@ -91,9 +91,15 @@ public function activate(): ScopeInterface
 
     private static function debugScopesDisabled(): bool
     {
-        $disabled = $_SERVER['OTEL_PHP_DEBUG_SCOPES_DISABLED'] ?? ini_get('OTEL_PHP_DEBUG_SCOPES_DISABLED');
+        if (isset($_SERVER['OTEL_PHP_DEBUG_SCOPES_DISABLED']) && filter_var($_SERVER['OTEL_PHP_DEBUG_SCOPES_DISABLED'], FILTER_VALIDATE_BOOLEAN)) {
```

Comment:
since the `OTEL_PHP_DEBUG_SCOPES_DISABLED` string is used a lot, can you move it into a class constant?